tests/dut_console/test_idle_timeout.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import pexpect
+import logging
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_TMOUT = "900"
+SET_TMOUT = "10"
+
+def test_timeout(duthost_console, duthost):
+    logger.info("Get default session idle timeout")
+    default_tmout = duthost_console.send_command('echo $TMOUT')
+    pytest_assert(default_tmout == DEFAULT_TMOUT, "default timeout on dut is not 900 seconds")
+
+    logger.info("Set session idle timeout")
+    duthost_console.send_command('export TMOUT={}'.format(SET_TMOUT))
+    set_tmout = duthost_console.send_command('echo $TMOUT')
+    pytest_assert(set_tmout == SET_TMOUT, "set timeout fail")
+
+    time.sleep(15)
+    duthost_console.send_command("\n", expect_string=r"str-s6000-on-6 login:")
```

Comment:
Is the test case designed for only "str-s6000-on-6" ?

Reply:
sorry....I forget to change this...